nodes/viz/texture_viewer.py

```diff
@@ -152,9 +152,17 @@ def draw_buttons(self, context, l):
         c.prop(self, 'activate')
 
     def draw_buttons_ext(self, context, l):
+        l.label(text="choose a different color for the border:")
         l.prop(self, "selected_theme_mode")
+        l.separator()
         l.label(text="save texture as bitmap image, choose a format:")
         l.prop(self, "bitmap_save")
+        row = l.row()
+        #addon = context.user_preferences.addons.get(sverchok.__name__)
+        #row.scale_y = 4.0 if addon.preferences.over_sized_buttons else 1
+        opera = row.operator('save_bitmap', text="S A V E")
```

Comment:
This is wrong because the operator neeed an operator, right? 
looking in viewer_mk2:
how should solve this? should i create another class inherited to bpy.types.Operator?